gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/V2CanaryController.groovy

```diff
@@ -54,7 +55,23 @@ class V2CanaryController {
     v2CanaryService.listJudges()
   }
 
-  // TODO: Add endpoint for initiating a canary run.
+  @ApiOperation(value = 'Start a canary execution')
+  @RequestMapping(value = '/canary/{canaryConfigId:.+}', method = RequestMethod.POST)
```

Comment:
i'm not sure whether or not i actually need the `:.+`, but it was on the path matcher for the kayenta endpoint itself so i kept it.